fbxosctrl.py

```diff
@@ -476,8 +562,11 @@ def __init__(self, controller):
         self.cmdCallbacks = {
             'regapp': self.controller.registerApp,
             'wifistatus': self.controller.getWifiStatus,
-            'wifion': self.controller.setWifiOn,
-            'wifioff': self.controller.setWifiOff,
+            'woff': self.controller.setWifiOn,
```

Comment:
I suggest to use 'wron' and  'wroff' meaning 'wifi radio ON' and 'wifi radio OFF' respectively (instead of 'won'/'woff').
To have common pattern but differentiated from wifi planning 'wpon'/'wpoff'.